examples/medplum-task-example-app/app/src/components/task-actions/AddNote.tsx

```diff
@@ -0,0 +1,114 @@
+import { Button, Modal } from '@mantine/core';
+import { notifications } from '@mantine/notifications';
+import {
+  createReference,
+  getQuestionnaireAnswers,
+  MedplumClient,
+  normalizeErrorString,
+  PatchOperation,
+} from '@medplum/core';
+import { Annotation, Questionnaire, QuestionnaireResponse, Task } from '@medplum/fhirtypes';
+import { QuestionnaireForm, useMedplum } from '@medplum/react';
+import { IconCircleCheck, IconCircleOff } from '@tabler/icons-react';
+import { useState } from 'react';
+
+interface AddCommentProps {
+  task: Task;
+  onChange: (updatedTask: Task) => void;
+  medplum: MedplumClient;
+}
+
+export function AddNote(props: AddCommentProps): JSX.Element {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const author = props.medplum.getProfile();
```

Comment:
we also have a hook called `useMedplumProfile()`, which is a custom hook to get what you need